apps/ensadmin/src/components/ensnode/hooks.ts

```diff
@@ -78,18 +78,6 @@ function validateResponse(response: EnsNode.Metadata) {
         .join(", ")}`,
     );
   }
-
-  const networksWithoutLastIndexedBlock = Object.entries(networkIndexingStatusByChainId).filter(
-    ([, network]) => network.lastIndexedBlock === null,
-  );
-
-  if (networksWithoutLastIndexedBlock.length > 0) {
-    throw new Error(
-      `Missing last indexed block for some networks with the following chain IDs: ${networksWithoutLastIndexedBlock
-        .map(([chainId]) => chainId)
-        .join(", ")}`,
-    );
-  }
```

Comment:
There's a legit case for a chain not to include the last indexed block value. For example, the chain might be waiting for its turn to be indexed. For the time being, I'll make the viewmodel responsible for mapping proper UI output.